v2/tools/generator/internal/codegen/pipeline/apply_export_filters.go

```diff
@@ -32,47 +31,69 @@ func filterTypes(
 	configuration *config.Configuration,
 	state *State) (*State, error) {
 
-	newDefinitions := make(astmodel.Types)
-
-	filterer := configuration.BuildExportFilterer(state.types)
```

Comment:
I don't have any issue with us deleting this, BUT this `filterTypes` is doing something quite different than it was before.

Before: It applied filters to all types, and could manually exclude things _or_ include things.
Now: It applies filters only to resource types and excludes everything by default unless it was manually included. There's no way to include everything in a package. I'm ok with this (we can expand it back out to support that in the future if we want).

I think it's clearer now, but we have a bunch of rules that weren't `include-transitive`, like:
```
  - action: exclude
    group: definitions
    name: '*ResourceBase*'
    because: Not actually a resource, used for inheritance only.
  - action: exclude
    group: deploymenttemplate
    name: '*ResourceBase*'
    because: Not actually a resource, used for inheritance only.
```

these are still in the YAML but as far as I can tell they aren't doing anything?